workflow/rules/simulation.smk

```diff
@@ -0,0 +1,71 @@
+# Copyright 2025 Xin Huang
+#
+# GNU General Public License v3.0
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program. If not, please see
+#
+#    https://www.gnu.org/licenses/gpl-3.0.en.html
+
+
+rule simulate_neutral:
+    output:
+        src_vcf = "results/simulated_data/run{run}/src.run{run}.vcf",
+        tgt_vcf = "results/simulated_data/run{run}/tgt.run{run}.vcf", 
+        ref_vcf = "results/simulated_data/run{run}/ref.run{run}.vcf",
+        final_vcf = "results/simulated_data/run{run}/simulation.run{run}.vcf"
+    params:
+        run = "{run}"
+    shell:
+        """
+        sed 's|p1.outputVCFSample(1);|p1.outputVCFSample(1, filePath="{output.src_vcf}");|; s|p2.outputVCFSample(10);|p2.outputVCFSample(10, filePath="{output.tgt_vcf}");|; s|p3.outputVCFSample(10);|p3.outputVCFSample(10, filePath="{output.ref_vcf}");|' workflow/scripts/racimo2017_neutral.slim > /tmp/neutral_{params.run}.slim
```

Comment:
Looks like the `filePath` parameter complicates the procedure. Do you have any idea to make it simpler?